v3/util/alt_reg_num_ev.go

```diff
@@ -0,0 +1,76 @@
+package util
+
+/*
+* ZLint Copyright 2025 Regents of the University of Michigan
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy
+* of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+* implied. See the License for the specific language governing
+* permissions and limitations under the License.
+ */
+
+import (
+	"github.com/zmap/zcrypto/encoding/asn1"
+)
+
+type RDNSequence []RelativeDistinguishedNameSET
+
+type RelativeDistinguishedNameSET []AttributeTypeAndValue
+
+type AttributeTypeAndValue struct {
+	Type  asn1.ObjectIdentifier
+	Value asn1.RawValue
+}
+
+type parsedSubjectElement struct {
+	IsPresent    bool
+	Value        string
+	Asn1RawValue asn1.RawValue
+	ErrorString  string
+}
+
+type ParsedEvOrgId struct {
+	Rsi, Country, StateOrProvince, RegRef string
+}
+
+func GetSubjectOrgId(rawSubject []byte) parsedSubjectElement {
```

Comment:
I'm surprised that the linter isn't complaining that a public function is returning a private type. :shrug: I can't say that I'm a huge fan of golangci.